src/payment/strategies/cba-mpgs/cba-mpgs-payment-strategy.ts

```diff
@@ -180,11 +180,14 @@ export default class CBAMPGSPaymentStrategy extends CreditCardPaymentStrategy {
                 return reject(new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized));
             }
 
-            this._threeDSjs.authenticatePayer(orderId, transactionId, data => {
+            this._threeDSjs.authenticatePayer(orderId, transactionId, async data => {
                 const error = data.error;
 
                 if (error) {
                     if (error.cause && error.cause === THREE_D_SECURE_BUSY && attempt < 5) {
+                        // Wait 2 seconds for MPGS server to process the `initiateAuthentication` call
```

Comment:
Is there a document that this can be linked to? Or some other kind of reference.

Reply:
If it says to wait for 10 seconds as in description, does it make sense to increase the wait time higher to 4 or so. So we don't hammer the API that much?

Reply:
@animesh1987 I've added a Link to the documentation.
Also I increased the delay time to 3 seconds, which gives a margin in the case the process finishes sooner (which happens on the test environment) or in case it finishes in time there's still 2 retries pending without waiting any longer.